### DISCUSSION TITLE
Abp Framework Mvc Web App Token

### DISCUSSION THREAD

**Discussion by burakdmir** [abpframework/abp#17492] [Q&A] [answered]
Hi,

I have developed an application using MVC. I need to obtain a token for communication with a mobile application, and I am having trouble with this process. Can you guide me on which endpoints and parameters to use for token acquisition? I am using the latest version of ABP Framework. The addresses and parameters I'm using are provided below.

/connect/authorize

grant_type:authorization_code
client_id:S****
client_secret:1q2w3e*
scope:app_name profile email address roles openid
response_type:code
redirect_uri:https://localhost:44366/signin-oidc
nonce:qerterw
prompt:none
username:****
password:*****

**Comment by maliming** [comment]
hi

You can get an access token by password grant.

https://oauth.net/2/grant-types/password/

**Reply by burakdmir** [reply]
![Ekran görüntüsü 2023-08-28 091500](https://github.com/abpframework/abp/assets/20765936/09f97e2a-9903-47a8-aa8d-0ad79a7075c7)

**Reply by maliming** [reply, marked as answer]
This belongs OAuth2 scope. You can Google it.

https://github.com/abpframework/abp/blob/dev/modules/openiddict/app/OpenIddict.Demo.Client.Console/Program.cs#L22
